### DISCUSSION TITLE
How to call functions defined on Setup script?

### DISCUSSION THREAD

**Discussion by adoin** [vuejs/devtools-v6#1978] [Q&A]
For Vue 2 we can inspect vue component then $vm0.functionName() to call that function.
Now I can still get $vm0 and props or states, but cant call function any more.
How or is not impletment?
Any Ideas?

**Comment by adoin** [comment]
Oh I get it from another question, leave it...

**Reply by Phunky** [reply]
Care to link to the question :D I also want to know how to access component instances again via `$vm0` for things like `$store` and methods.

**Reply by adoin** [reply]
$vm0.proxy to get the vue page context. then if you have assign getter there, you can get whatever you want include $store.

**Reply by Phunky** [reply]
Literally stumbled on [this discussion](https://github.com/vuejs/devtools/discussions/1606) after posting this comment :D but thank you for the detail. I'm sure that'll help others that land here from a search 👍

**Comment by adoin** [comment]
Discussion is the best teacher.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mark ***@***.***&gt;; 
发送时间: 2022年10月26日(星期三) 下午5:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [vuejs/devtools] How to call functions defined on Setup script? (Discussion #1978)





 
Literally stumbled on this discussion after posting this comment :D but thank you for the detail. I'm sure that'll help others that land here from a search 👍
 
&mdash;
